docs/quick-start/transmitters/lua-howto.md

```diff
@@ -56,28 +53,52 @@ The `Connection Status` indicator shows your current RC link status. When not co
+	- `D250Hz & D500Hz` :new: : These are FLRC-based options. `D` stands for `Déjà Vu`. See [PR1527](https://github.com/ExpressLRS/ExpressLRS/pull/1527) for the technical details.
+	- `100Hz Full & 333Hz Full` :new: : These are Lora-based Full resolution modes. See [PR1572](https://github.com/ExpressLRS/ExpressLRS/pull/1572) for more technical details.
+	
+	The following options are available for `900MHz`:
+
+	- `25, 50, 100 & 200`: These are the LoRa-based options we started with.
+	- `100Hz Full` :new: : Full resolution mode for Team900. See [PR1572](https://github.com/ExpressLRS/ExpressLRS/pull/1572) for more technical details.
+
+	The number following the rate in parentheses (e.g. -105dBm for 500Hz) is the Sensitivity Limit for the rate, the lowest RSSI where packets will still be received. See [Signal Health](../../info/signal-health.md) for more information about the sensitivity limit.
 
 !!! warning "WARNING"
 	NEVER change the packet rate while flying as this FORCES A DISCONNECT between the TX and RX. 
 
-* `Telem Ratio` sets the telemetry ratio, how much of the packet rate is used to send telemetry. The options, in order of increasing telemetry rate, are: `Off, 1:128, 1:64, 1:32, 1:16, 1:8, 1:4, 1:2`. A Telem Ratio of 1:64 means one out of every 64 packets are used for telemetry data. For information on telemetry setup, see [First Flight: Telemetry](../pre-1stflight.md#telemetry) and [Telemetry Bandwidth](../../info/telem-bandwidth.md).
+* `Telem Ratio` sets the telemetry ratio or how much of the packet rate is used to send telemetry. The options, in order of increasing telemetry rate, are: `Off, 1:128, 1:64, 1:32, 1:16, 1:8, 1:4, 1:2`. A Telem Ratio of 1:64 means one out of every 64 packets are used for telemetry data. For information on telemetry setup, see [First Flight: Telemetry](../pre-1stflight.md#telemetry) and [Telemetry Bandwidth](../../info/telem-bandwidth.md).
+	- :new: v3.0 also comes with `Std` and `Race` options. `Std` changes depending on the Packet Rate, while `Race` will give you Telemetry while not yet armed, stopping Telemetry once armed.
 
 ### Switch Mode
 
 The Switch Mode setting controls how channels AUX1-AUX8 are sent to the receiver (the 4 main channels are always 10-bit). The options are `Hybrid & Wide`. **Hybrid** mode is 6x 2/3/6-position + 1x 16-position, and **Wide** is 7x 64 or 128-position. For detail about the differences, see the [Switch Configs](../../software/switch-config.md) documentation.
 
+:new: With the introduction of the Full Resolution modes, new Switch Modes can now be set for them. **8CH Mode** has CH1-CH4(sticks) and CH6-CH9 sent as 10-bit (Full Resolution) at the selected rate, with CH5(Aux1) remaining as 1-bit for Arming state. **12CH Mode** has CH1-CH4(sticks) sent as 10-bit along with CH5(Aux1) sent as 1-bit at the selected rate, and CH6-CH13 sent as 10-bit at half rate. **16CH Mode** has all channels (CH1-CH16) sent at 10-bit at half rate.
+
```

Comment:
:new: In full resolution modes, the Switch Mode parameter selects the number of channels to be used. [Full Resolution Switch Modes] 

Fix that link to the section below that explains it, don't explain it in both places.